airbyte-server/src/main/java/io/airbyte/server/handlers/WebBackendConnectionsHandler.java

```diff
@@ -128,20 +126,18 @@ private WebBackendConnectionRead buildWebBackendConnectionRead(final ConnectionR
     final SourceRead source = getSourceRead(connectionRead);
     final DestinationRead destination = getDestinationRead(connectionRead);
     final OperationReadList operations = getOperationReadList(connectionRead);
-    final JobReadList syncJobReadList = getSyncJobs(connectionRead);
+    final Optional<JobRead> latestRunningSyncJob = jobHistoryHandler.getLatestRunningSyncJob(connectionRead.getConnectionId());
 
     final WebBackendConnectionRead webBackendConnectionRead = getWebBackendConnectionRead(connectionRead, source, destination, operations)
-        .catalogId(connectionRead.getSourceCatalogId())
-        .isSyncing(syncJobReadList.getJobs()
-            .stream()
-            .map(JobWithAttemptsRead::getJob)
-            .anyMatch(WebBackendConnectionsHandler::isRunningJob));
-    setLatestSyncJobProperties(webBackendConnectionRead, syncJobReadList);
-    return webBackendConnectionRead;
-  }
+        .catalogId(connectionRead.getSourceCatalogId());
+
+    if (latestRunningSyncJob.isPresent()) {
```

Comment:
Nit for later:   could do `latestRunningSyncJob.ifPresent( -> { //set the fields here })` to avoid the if check.